Person.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Person.swift
+//  nameToFaces
+//
+//  Created by Felipe Gil on 2021-06-21.
+//
+
+import UIKit
+
+class Person: NSObject {
```

Comment:
```suggestion
struct Person {
```

We usually don't create classes for models. Instead, we use structs, which are value types that are copied when passed into other objects. In your case, the Person values are stored in your ViewController and are not passed anywhere, so a struct is more suitable here. Also, structs are not inheritable.

Other notes: when creating a class, it doesn't need to inherit from `NSObject`. There are cases you'll need that because of Objective-C compatibility, but the general rule of thumb is you don't need it. In case you do need to inherit from `NSObject`, you don't need to import the entire `UIKit` framework, but only the `Foundation` framework (it's working in your case because `UIKit` imports Foundation).

Here is a simplified article about it classes and structs: https://medium.com/quick-code/class-vs-struct-in-swift-f8c48eaacdba. If you want to go deeper (and you'll probably want to do that in the future), here is the official reference, the Swift book: https://docs.swift.org/swift-book/LanguageGuide/ClassesAndStructures.html.

Reply:
That's right. We had a chance to talk about it. 
For this particular part, I followed Paul's code, but I had no idea about the downside of using classes for this solution. 
Thank you, I've changed already.